awslabs/generative-ai-cdk-constructs
# chore(deps): update langchain deps

Fixes https://github.com/awslabs/generative-ai-cdk-constructs/security/dependabot 

Testing:

Ingestion 

<img width="675" alt="Screenshot 2024-08-06 at 10 55 40 AM" src="https://github.com/user-attachments/assets/3b0cb087-dd90-4d08-8796-f357cf563db3">

QA

RAG:
<img width="1473" alt="Screenshot 2024-08-06 at 10 56 38 AM" src="https://github.com/user-attachments/assets/b6ce3a33-e5b9-41c4-8376-a1a94fdb0de3">

Long context:
<img width="1486" alt="Screenshot 2024-08-06 at 10 57 26 AM" src="https://github.com/user-attachments/assets/d4c6d30d-98f7-49f2-ade1-64fd074466a7">

Summarization:

<img width="657" alt="Screenshot 2024-08-06 at 10 59 36 AM" src="https://github.com/user-attachments/assets/39e8a178-9db7-448b-b067-f8fc40313053">


---
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the project license.
